development/probes/probe-k8s-metrics-server/src/main/java/eubr/atmosphere/tma/probes/ProbeKubernetes.java

```diff
@@ -96,14 +98,15 @@ public static void main(String[] args) {
                 response = requestRestService(uriNodes);
                 isr = new InputStreamReader(response.getEntity().getContent());
                 parseNodeMetrics(isr, client);
-                Thread.sleep(60000);
+
+                // This is the frequency that the probe collect the data and send to the monitor
+                Thread.sleep(5000);
```

Comment:
Use a construction like below, as it is more self explanatory               
TimeUnit.SECONDS.sleep(5);